javascript/apps/taiga/src/app/shared/comments/components/deleted-comment/deleted-comment.component.ts

```diff
@@ -0,0 +1,107 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * Copyright (c) 2023-present Kaleidos INC
+ */
+
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Input,
+  OnInit,
+} from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { UserComment } from '@taiga/data';
+import { User } from '@taiga/data';
+import { RxState } from '@rx-angular/state';
+import {
+  TRANSLOCO_SCOPE,
+  TranslocoModule,
+  TranslocoService,
+} from '@ngneat/transloco';
+import { Store } from '@ngrx/store';
+import { filterNil } from '~/app/shared/utils/operators';
+import { selectUser } from '~/app/modules/auth/data-access/+state/selectors/auth.selectors';
+import { DateDistancePipe } from '~/app/shared/pipes/date-distance/date-distance.pipe';
+
+@Component({
+  selector: 'tg-deleted-comment',
+  standalone: true,
+  templateUrl: './deleted-comment.component.html',
+  styleUrls: ['./deleted-comment.component.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  imports: [CommonModule, TranslocoModule, DateDistancePipe],
+  providers: [
+    RxState,
```

Comment:
Probably you don't need RxState

Reply:
You are not using rxstate in this file, delete it